File: rentabikewtr_desktop/lib/model/poziviDezurnomVoziluPregled.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'poziviDezurnomVoziluPregled.g.dart';

@JsonSerializable()
class PoziviDezurnomVoziluPregled {
  int? pozivDezurnomVoziluId;
  String? viseDetalja;
  bool? nesreca = false;
  bool? zahtjevKlijenta = false;
  bool? kvar = false;
  bool? losiVremenskiUslovi = false;
  DateTime? datumPoziva;
  DateTime? vrijemePoziva;
  String? nazivDezurnogVozila;
  String? naziv; // Naziv turistickog vodica
  String? nazivPoslovnice;

  PoziviDezurnomVoziluPregled() {}

  factory PoziviDezurnomVoziluPregled.fromJson(Map<String, dynamic> json) =>
      _$PoziviDezurnomVoziluPregledFromJson(json);

  /// Connect the generated [_$PoziviDezurnomVoziluPregledToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PoziviDezurnomVoziluPregledToJson(this);
}

File: RentABikeWTR_v1_7.Model/PoziviDezurnomVoziluPregled.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RentABikeWTR_v1_7.Model
{
    public class PoziviDezurnomVoziluPregled
    {
        
        public int? PozivDezurnomVoziluId { get; set; }
        public String? ViseDetalja { get; set; }
        public bool? Nesreca { get; set; }
        public bool? ZahtjevKlijenta { get; set; }
        public bool? Kvar { get; set; }
        public bool? LosiVremenskiUslovi { get; set; }
        public DateTime? DatumPoziva { get; set; }
        public DateTime? VrijemePoziva { get; set; }
        public int? DezurnoVoziloID { get; set; }
        public String? NazivDezurnogVozila { get; set; }
        public int? TuristickiVodicID { get; set; }
        public String? Naziv { get; set; } // Naziv turistickog vodica
        public int? PoslovnicaID { get; set; }
        public String? NazivPoslovnice { get; set; }
    }
}
